frontend/packages/console-app/src/status/node.ts

```diff
@@ -1,4 +1,5 @@
 import { NodeKind } from '@console/internal/module/k8s';
-import { isNodeReady } from '@console/shared/src/selectors/node';
+import { isNodeReady, getCpuArch } from '@console/shared/src/selectors/node';
 
 export const nodeStatus = (node: NodeKind) => (isNodeReady(node) ? 'Ready' : 'Not Ready');
+export const getNodeCpuArch = (node: NodeKind) => getCpuArch(node);
```

Comment:
I don't think we need a new shared selector here:
```suggestion
export const getNodeArchitecture = (node: NodeKind) => {
  return isCSRResource(node) ? 'amd64' : node?.status?.nodeInfo?.architecture;
}
```